CMakeModules/CompilerSettings.cmake

```diff
@@ -77,11 +77,19 @@ if((CMAKE_COMPILER_IS_GNUCXX OR IS_ICPC) AND NOT MINGW)
     endif()
 endif()
 
+# By default CMake sets RPATH for binaries in the build tree, but clears
+# it when installing. Switch this option off if the default behaviour is
+# desired.
+option(NO_DEFAULT_RPATH "Set RPATH for installed binaries" ON)
+
 # Set rpath http://www.paraview.org/Wiki/CMake_RPATH_handling
-set(CMAKE_SKIP_BUILD_RPATH FALSE)
-set(CMAKE_BUILD_WITH_INSTALL_RPATH FALSE)
-set(CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_LIBDIR_FULL}")
-set(CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE)
+if(NO_DEFAULT_RPATH)
+    message(STATUS "Set RPATH explicitly to '${CMAKE_INSTALL_LIBDIR_FULL}'")
+    set(CMAKE_SKIP_BUILD_RPATH FALSE)
+    set(CMAKE_BUILD_WITH_INSTALL_RPATH FALSE)
+    set(CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_LIBDIR_FULL}")
+    set(CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE)
+endif(NO_DEFAULT_RPATH)
```

Comment:
The style used in fcl's CMakeLists (at least in this one) is to avoid keeping the condition in the `endif()`, I would suggest removing this.